### ISSUE TITLE
Occasional refresh in Chrome and Firefox on window resize

### ISSUE THREAD

**Issue by lewelyngraceaquino** [ozoneplatform/owf-framework#91]
We are using OWF Version: 7.17.1 Build Date: January 16 2017 Build Number: 2 and we have noticed that at times, when we resize the browser window it reloads/refreshes the browser and resets all the opened widgets losing any changes made to them and bringing them back to their initial state. 

This is causing issues especially when demoing our application to users as it can then take up to 5 minutes to get back on track and is an extremely embarrassing look. This has been tested in both Chrome and Firefox.

I have installed a fresh bundle (Version: 7.17.1 Build Date: January 16 2017 Build Number: 2) and just used the sample widgets that came with it and I can recreate the issue in Chrome and Firefox too.

Is this the default behaviour or is this a bug?

**Comment by DexterHaslem** [comment]
just wanted to let you know that this has been around for a while,  I've been fighting it for the last several versions as well

**Comment by andrew-dwyer** [comment]
Hi @DexterHaslem, have you figured out any work arounds or solutions?

**Comment by DexterHaslem** [comment]
No, just making sure to not resize in-front of clients. I do intend to look at root cause at some point. If I do find a fix I will share here (even though this repo is somewhat disregarded in the PR department)
